sc-extensions/src/main/java/com/powsybl/sc/extensions/TwoWindingsTransformerFortescuePartOfGu.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2022, Jean-Baptiste Heyberger & Geoffroy Jamgotchian
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.sc.extensions;
+
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.TwoWindingsTransformer;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at gmail.com>
+ */
+public class TwoWindingsTransformerFortescuePartOfGu extends AbstractExtension<TwoWindingsTransformer> {
```

Comment:
Do you really need to create this extension ? We have in powsybl-core and extension called `GeneratorShortCircuit`. In it, we have the method `getStepUpTransformerX`. We can add the boolean you need in this extension, or maybe just rely on a non-zero value of the reactance of the step up transformer ?  In any case, it is not releated to `Foretescue`, so the extension name has to be changed.

Reply:
After a second thought:  I think we have to use the core extension, but in the mind time you can introduce a property called `hasStepUpTransformer` for example.